Common/Global/BossSummonOverride.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+
+namespace TheBereftSouls.Common.Global
+{
+    public class BossSummonOverride : GlobalItem
+    {
+        public override void SetDefaults(Item entity)
+        {
+            base.SetDefaults(entity);
+        }
```

Comment:
Where's this method in `Terraria.ModLoader.GlobalItem`?

I can't find it in the [docs](https://docs.tmodloader.net/docs/preview/class_global_item.html) nor the [source code](https://github.com/tModLoader/tModLoader/blob/1.4.4/patches/tModLoader/Terraria/ModLoader/GlobalItem.cs).

Reply:
my bad